ImageKitSample/Program.cs

```diff
@@ -0,0 +1,487 @@
+            ob3.Add("fileId_1");
+            ob3.Add("fileId_2");
+            ResultFileDelete resultFileDelete = imagekit.BulkDeleteFiles(ob3);
+
+            // Copy File
+            CopyFileRequest cpyRequest = new CopyFileRequest
+            {
+                sourceFilePath = "path_1",
+                destinationPath = "path_2"
+            };
+            ResultNoContent resultNoContent = imagekit.CopyFile(cpyRequest);
+
+            // MoveFile
+            MoveFileRequest moveFile = new MoveFileRequest
+            {
+                sourceFilePath = "path_1",
+                destinationPath = "path_2"
+            };
+            ResultNoContent resultNoContentMoveFile = imagekit.MoveFile(moveFile);
+
+            // RenameFile
+            RenameFileRequest renameFileRequest = new RenameFileRequest
+            {
+                filePath = "path_1",
+                newFileName = "file_name",
+                purgeCache = false
+            };
+            ResultRenameFile resultRenameFile = imagekit.RenameFile(renameFileRequest);
+
+            #endregion
+
+            #region  Tags
+            TagsRequest tagsRequest = new TagsRequest
+            {
+                tags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2"
+                },
+                fileIds = new List<string>
+                {
+                    "fileId_1",
+                },
+            };
+            ResultTags resultTags = imagekit.AddTags(tagsRequest);
+
+            TagsRequest removeTagsRequest = new TagsRequest
+            {
+                tags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2"
+                },
+                fileIds = new List<string>
+                {
+                    "fileId_1",
+                },
+            };
+            ResultTags removeTags = imagekit.RemoveTags(removeTagsRequest);
+
+            AITagsRequest removeAITagsRequest = new AITagsRequest
+            {
+                AITags = new List<string>
+                {
+                    "tag_1",
+                    "tag_2"
+                },
+                fileIds = new List<string>
+                {
+                    "fileId_1",
+                },
+            };
+            ResultTags removeAITags = imagekit.RemoveAITags(removeAITagsRequest);
+
+            #endregion
+
+            #region FileVersionRequest
+            DeleteFileVersionRequest delRequest = new DeleteFileVersionRequest
+            {
+                fileId = "file_Id",
+                versionId = "version_Id"
+            };
+            ResultNoContent resultNoContent1 = imagekit.DeleteFileVersion(delRequest);
+
+            // RestoreFileVersion
+            Result result = imagekit.RestoreFileVersion("file_Id", "version_Id");
+
+            // GetFileVersions
+            ResultFileVersions resultFileVersions = imagekit.GetFileVersions("file_Id");
```

Comment:
```
Unhandled exception. System.Exception: Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'ResultFileVersions' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.
To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.
Path '', line 1, position 1.
   at Imagekit.Helper.ImagekitException..ctor(String message)
   at RestClient.GetFileVersions(String fileId)
   at Imagekit.Sdk.ImagekitClient.GetFileVersions(String fileId)
```